draft-ietf-quic-http.md

```diff
@@ -410,10 +417,10 @@ considered invalid.
 
 An HTTP request/response exchange fully consumes a bidirectional QUIC stream.
```

Comment:
I guess while we are changing so much text we could make it explicity and say

"An HTTP request/response exchange fully consumes a client-initiated bidirectional QUIC stream."

Reply:
Good call.  Yes, this covers pushes, too -- it's a response message that follows this format.  (I don't see *why* you'd push with a non-final response in the sequence, but it's totally valid....)

Reply:
After reading the whole section, I think that while the section intends to talk about both push and requests, this paragraph is specific to requests.  I've added more discussion of push streams above, but I think I'm going to leave this particular paragraph alone, modulo fixing @LPardue's initial suggestion.